core/src/main/java/org/opensearch/sql/expression/function/BuiltinFunctionName.java

```diff
@@ -58,6 +58,14 @@ public enum BuiltinFunctionName {
   TAN(FunctionName.of("tan")),
   SPAN(FunctionName.of("span")),
 
+  /** Collection functions */
+  ARRAY(FunctionName.of("array")),
```

Comment:
do you miss `ARRAY_LENGHT`? https://github.com/opensearch-project/opensearch-spark/blob/main/docs/ppl-lang/functions/ppl-collection.md#array

Reply:
Already add it.